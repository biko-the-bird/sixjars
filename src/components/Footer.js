import React from 'react';

const Footer = () => {
    return (
        <div className="footer-container">
            <p>Copyright Â© 2019 Zachary Elardo</p>
        </div>
    )
}

export default Footer;