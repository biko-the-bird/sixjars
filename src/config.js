
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

//this is a configuration for enzyme that needs
//to be imported in each testing file
const config = configure({ adapter: new Adapter() });
export default config;