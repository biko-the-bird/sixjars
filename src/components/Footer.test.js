import React from 'react';
import {shallow} from 'enzyme';
import config from '../config';
import Footer from './Footer';


describe("footer", () => {
    const footer = shallow(<Footer/>);
    it("renders", () => {
        expect(footer).toMatchSnapshot();
    });
   
})