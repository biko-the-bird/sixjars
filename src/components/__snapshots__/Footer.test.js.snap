// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer renders 1`] = `ShallowWrapper {}`;
